packages/core/src/components/Links/_Links.docs.scss

```diff
@@ -0,0 +1,51 @@
+/*
+Links
+
+Links connect users to a different page or further information.
+
+@uswds https://standards.usa.gov/components/labels/
```

Comment:
As I understand, USWDS doesnt have a link component, this links to https://designsystem.digital.gov/components/tag/, which is entirely different. Much of the guidance on this page is also from Tags

Reply:
USWDS places their link information under typography https://designsystem.digital.gov/components/typography/#links